data/operators/amenity/car_sharing.json

```diff
@@ -479,21 +480,33 @@
       }
     },
     {
-      "displayName": "RTTB",
-      "id": "rttb-672a50",
-      "locationSet": {"include": ["fx"]},
+      "displayName": "Scouter",
+      "id": "scouter-088af7",
+      "locationSet": {"include": ["de"]},
       "tags": {
         "amenity": "car_sharing",
-        "operator": "RTTB"
+        "operator": "Scouter"
       }
     },
     {
-      "displayName": "Scouter",
-      "id": "scouter-088af7",
-      "locationSet": {"include": ["de"]},
+      "displayName": "Share Now",
+      "id": "sharenow-6a8c71",
+      "locationSet": {
+        "include": [
+          "at",
+          "ca",
+          "de",
+          "dk",
+          "fx",
+          "it",
+          "nl",
+          "us"
```

Comment:
[The company’s official city page](https://www.share-now.com/de/en/country-list/) only lists eight European countries; it looks like they’ve pulled out of North America.

Reply:
indeed, I did take the list from the [English Wikipedia page](https://en.wikipedia.org/wiki/Share_Now) but it seems outdated. "ca" and "us" will be removed in next commit.